.github/workflows/cd.yml

```diff
@@ -0,0 +1,72 @@
+name: CD
+on:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  PY_VERSION: 3.12
+
+jobs:
+  pypi-build:
+    name: Build package for PyPI
+    if: github.repository == 'ACCESS-NRI/access-parsers'  # exclude forks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ env.PY_VERSION }}
+
+      - run: |
+          python3 -m pip install --upgrade build && python3 -m build
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: release
+          path: dist
+
+  pypi-publish:
+    # Split build and publish to restrict trusted publishing to just this workflow
+    needs: ['pypi-build']
+    name: Publish to PyPI.org
+    runs-on: ubuntu-latest
+    permissions:
+      # IMPORTANT: this permission is mandatory for trusted publishing
+      id-token: write
```

Comment:
You'll need to add this project as a "pending publisher" on PyPI if the project doesn't already exist as a trusted publisher. Happy to help with this

Reply:
Here are some instructions for adding a PyPI project with "pending" trusted publishing: https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/

Login details for the accessnri PyPI account are in Bitwarden in the Model Release Collection